src/test/ui/parser/issue-105634.rs

```diff
@@ -0,0 +1,8 @@
+// run-pass
```

Comment:
Since this is testing a parser issue it only needs to be `check-pass`.

Reply:
Sure!